ols/app/models/config.py

```diff
@@ -113,6 +114,11 @@ class ModelConfig(BaseModel):
 
     options: Optional[dict[str, Any]] = None
 
+    # model_* is protected namespace by default, as we want to use
+    # `model_parameters`, we need to remove it from protected namespaces
```

Comment:
given the class name is ModelConfig, could we just rename the field to `parameters` instead?  

i don't think it matters a whole lot, but erasing all these warnings seems like overkill and may cause confusion in the future if we shadow/override something that matters?

Reply:
Updated.